website/docs/reference/resource-configs/delimiter.md

```diff
@@ -0,0 +1,86 @@
+---
+resource_types: [seeds]
+datatype: <string>
+default_value: ","
+---
+
+## Definition
+
+An optional seed configuration used to customize how you separate values in a [seed](/docs/build/seeds) with the one-character string you provide.
+
+* The delimiter defaults to a comma when not specified.
+* Explicitly set The `delimiter` configuration value if you want seed files to use a different delimiter, such as "|" or ";" (default is comma).
+
+:::info New in 1.7!
+
+Delimiter is new functionality available beginning with dbt Core v1.7.
+
+:::
+
+  
+## Usage
+
+Specify a custom delimiter in your `dbt_project.yml` file to override the global separator for all seed values:
+
+<File name='dbt_project.yml'>
+
+```yml
+seeds:
+  <project_name>:
+     +delimiter: "|"
+    <seed_directory_name>:
+      +delimiter: ","
+```
+
+</File>
+
+
+Or use a custom delimiter to override the values for a specific seed:
+
+<File name='seeds/properties.yml'>
+
+```yml
+version: 2
+
+seeds:
+  - name: <seed_directory_name>
```

Comment:
why do we have a seed named seed_director_name? should this just be seed_name?

Reply:
This works similarly to quote_columns, so we could follow the example pattern from that page https://docs-getdbt-com-git-delimiter-adding-dbt-labs.vercel.app/reference/resource-configs/quote_columns#only-quote-seeds-in-the-seedsmappings-directory

Reply:
Thanks @graciegoheen! I copied examples from the comments in the Core PR thread, so glad you cleaned them up! I will reference quote columns and glean form there!